js/ui/text_box/ui.text_editor.base.js

```diff
@@ -140,7 +147,9 @@ const TextEditorBase = Editor.inherit({
                     return isMaterial(themeName);
                 },
                 options: {
-                    stylingMode: config().editorStylingMode || 'filled'
+                    stylingMode: config().editorStylingMode || 'filled',
+                    label: '',
```

Comment:
Default value for `label` option is ''. Why do we need to redefine it for material?